README.md

```diff
@@ -40,6 +50,7 @@ cd portfolio
 - Twitter: https://twitter.com/cadivisea
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you format your social media links with proper markdown from the Microverse readme template. This will make them a clickable link
## EXAMPLE
```
👤 **David Isea**

- Twitter: [@cadivisea](https://twitter.com/cadivisea)
- LinkedIn: [david-isea-560424236](https://www.linkedin.com/in/david-isea-560424236/)
```